core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -88,7 +88,7 @@ UNWIND(	.fnstart)
 	 * PCR
 	 * - no change latency, enable clk gating
 	 */
-	mov_imm r0, 0x00004000
+	mov_imm r0, 0x00004800
```

Comment:
Value here doesn't match the comment above.

It would be preferable to construct this value from descriptive defines instead as we do in other files. Since content of this register is, as far as I know, Cortex A9 specific we could put those defines in `core/arch/arm/inlucde/arm32_cortex_a9.h`.